### PR TITLE
Social: File login

### PR DESCRIPTION
**Pull request by NitsJehangir**
write new details hers

### CODE REVIEW

File: hello.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (JehangirAhmad) [inline hello.txt]: Captilize H

**Review by JehangirAhmad — CHANGES_REQUESTED** [review]
Bring the changes

**Review by NitsJehangir — COMMENTED** [review]
Captilization done

**Review by JehangirAhmad — APPROVED** [review]
Everything looks good. Go for merge

**Pull request merged**